README.md

```diff
@@ -1,27 +1,163 @@
 # KubeVirt Enhancements Tracking and Backlog
 
-This repository is currently Work In Progress but will eventually be used to manage KubeVirt Enhancement Proposals (VEPs), emphasizing centralized prioritization and enhanced SIG involvement and collaboration.
+This repository is currently Work In Progress but will eventually be used to manage KubeVirt Enhancement Proposals (
+VEPs), emphasizing centralized prioritization and enhanced SIG involvement and collaboration.
 
-## Process (DRAFT)
+## WHY
 
-1. **VEP Creation**: VEP authors will initiate proposals via PRs to the `kubevirt/enhancements` repository. [Design proposal template](https://github.com/kubevirt/community/blob/main/design-proposals/proposal-template.md)
-2. **SIG Review and Collaboration**: Each VEP will have a target SIG, and the SIG will assign a dedicated reviewer to oversee the proposal, collaborate with other SIGs as needed, and provide feedback or veto when necessary.
-3. **Centralized Prioritization**: At the start of each release cycle, all accepted VEPs will be designated as the project’s priority, focusing community efforts on the associated pull requests. Acceptance will be based on community support and a commitment to implementation.
-4. **Visibility and Tracking**: The Author of an accepted VEPs will open an issue to track their progress, maturity stages (alpha, beta, GA), list the associated bugs, and user feedback
-5. **Single source of truth**: Each VEP will be the authoritative reference for the associated feature. This aligns with the Kubernetes KEP process. It will ensure that each enhancement
-Includes all the relevant information, including the design and the state.
+The process aims to focus the community's efforts on prioritized pull requests, increase the review bandwidth, and
+ensure clear visibility of feature progress and associated issues. The relatively new split into SIGs requires defining
+a common process to ensure synchronization between SIGs and uniformity of the project.
+
+**Glossary**
+
+- VEP
+- EF: Enhancement Freeze
+- CF: Code Freeze
+- RC: Release Candidate
+- Alpha/Beta
+- release team
+
+## Process (DRAFT 2nd iteration)
+
+1. **VEP Creation**: VEP authors will initiate proposals via PRs to the `kubevirt/enhancements` repository. 
+   See the [VEP template](https://github.com/kubevirt/enhancements/blob/main/veps/NNNN-vep-template/vep.md), 
+   each section needs to be filled even if the section doesn't apply to the VEP.
+
+2. **Visibility and Tracking**: The Author of VEP will open an issue to track their progress, maturity stages (alpha,
+   beta, GA), list the associated bugs, and user feedback.
+   See the [VEP issue template](https://github.com/kubevirt/enhancements/blob/main/.github/ISSUE_TEMPLATE/vep.md), 
+   each section should be filled.
+
+3. **SIG Review and Collaboration**: Each VEP will have one target SIG. In case of a cross SIG feature, the most relevant 
```

Comment:
```suggestion
3. **SIG Review and Collaboration**: Although each VEP can have multiple target SIGs, it needs to have a single owning SIG. In case of a cross SIG feature, the most relevant 
```